carbon-design-system/carbon
# fix(toggle): ensure toggle click target matches visual representation

Closes #12117

#### Changelog

**Changed**

- Changed the toggle's container element to render as `inline-block`
- Changed the toggle element to render as `inline-grid`

#### Testing / Reviewing

- Ensure the toggle receives a click event when clicking on either the toggle or the label (both, side and top labels)
- Ensure the toggle doesn't receive a click event outside of the above mentioned elements
